image_explorer/templates/html/image_explorer_edit.html

```diff
@@ -10,6 +10,13 @@
           </div>
           <span class="tip setting-help">{% trans "The title of the Image Explorer that is displayed to the user" %}</span>
       </li>
+      <li class="field comp-setting-entry is-set">
+          <div class="wrapper-comp-setting">
+            <input type="checkbox" class="setting-input edit-hotspot-coordinates-centered" id="edit_hotspot_coordinates_centered" {% if self.hotspot_coordinates_centered %}checked="checked"{% endif %} {% if self.hotspot_coordinates_centered %}disabled="disabled"{% endif %}>
+            <label class="label setting-label label-block" for="edit_hotspot_coordinates_centered">Image explore calculated by percentage to regain its center</label>
```

Comment:
Wrap the text with `{% trans %}` and please change it to: "Hotspot coordinates centered".